### ISSUE TITLE
Allow receiver and SubAccount wallet verification before initializing wallet to wallet chimoney transfer in API docs.

### ISSUE THREAD

**Issue by louis103** [Chimoney/chimoney-community-projects#182]
- [ ] Verify the origin and destination accounts if they exist in the chimoney database before initializing a wallet to wallet payments. This prevents money transfer to unauthorized accounts.

**Comment by Lymah123** [comment]
Hi @louis103 , do you mean how to guide?

**Comment by louis103** [comment]
No, I meant like have a way to verify if the sender and receiver are valid
in the database so as to prevent unauthorized transactions.

**Comment by Lymah123** [comment]
Okay.

**Comment by Lymah123** [comment]
Is it beginner friendly? @louis103

**Comment by louis103** [comment]
If you are good in API development then it will be easier to work on. Just
a simple code to check if the sender and receiver accounts exist on the
database before any new transaction.

**Comment by Lymah123** [comment]
Okay, I would like to give it a try.

Can I message you anytime incase I need some clarity?

**Comment by louis103** [comment]
Yes, of course you can.

**Comment by louis103** [comment]
Sorry, I think someone had already volunteered to take on this task. From
our previous conversation.

**Comment by Lymah123** [comment]
@louis103 , you can proceed with the task.

**Comment by louis103** [comment]
Ok, I will proceed with it.

**Comment by uchibeke** [comment]
Hi we handle this in our api. If sub account id isn’t valid, the transfer fails. Let me know if there’s something I’m missing
